.github/workflows/workflow.yml

```diff
@@ -366,7 +366,8 @@ jobs:
         dir="${CLOUDSDK_INSTALL_DIR:-${HOME}}/google-cloud-sdk"
         (
           . bin/_gcp.sh ;
-          install_gcloud_kubectl "$dir"
+          install_gcloud "$dir"
+          gcloud components install kubectl
```

Comment:
we have kubectl available via `bin/kubectl`, so this line is probably not necessary. (i had left it in `_gcp.sh` simply because i didn't want to change too many things when switching to github actions).

if you are calling scripts that expect `kubectl` in the path (like `bin/test-run` and `get_k8s_ctx`), we have this pattern:
https://github.com/linkerd/linkerd2/blob/9bb7b6f119fe3a78f48440241f01947ea3be2791/.github/workflows/workflow.yml#L174

Reply:
ah cool. I'll remove that then :+1: